src/select/select.js

```diff
@@ -289,16 +300,19 @@ class Select extends React.Component<PropsT, StatelessStateT> {
           />
         )}
         {selectedOptions.map(option => (
-          <Tag key={option.id} $multiple={multiple}>
+          <Tag key={option.id} $multiple={multiple} disabled={disabled}>
```

Comment:
this attribute will leak to DOM <img> tag, but it's not supported by this DOM element, that's why we name them `$disabled` if we want it to be ignored